src/components/pill/pill.stories.js

```diff
@@ -25,8 +25,10 @@ const getKnobs = (theme) => {
     children: text('children', 'Pill'),
     fill: boolean('fill', Pill.defaultProps.fill),
     onDelete: boolean('onDelete', false),
-    theme
+    theme,
+    size: select('size', OptionsHelper.sizesRestricted, Pill.defaultProps.size)
   };
+
   if (theme === THEMES.classic) {
```

Comment:
I know this isn't in your changes but if you will change something in this file, can you rewrite it to use `isClassic()` method please?

Reply:
updated.